lexer/theme-parser.y

```diff
@@ -999,8 +1006,9 @@ t_property_color
     $$.alpha = $8;
 }
 /** Hex colors parsed by lexer. */
-| T_COLOR {
+| T_COLOR t_property_color_opt_alpha_ws {
     $$ = $1;
+    $$.alpha  = $2;
```

Comment:
This overrides the alpha that is (possibly) already set.

maybe a `*=`  instead of `=`?